examples/WORKSPACE

```diff
@@ -5,6 +5,8 @@ http_archive(
     strip_prefix = "protobuf-master",
     urls = ["https://github.com/google/protobuf/archive/master.zip"],
 )
+load("@com_google_protobuf//:protobuf.bzl", "check_version")
```

Comment:
Can this be done in the BUILD file instead?

Reply:
No, `native.bazel_version` is only available in WORKSPACE-included skylark macros